migrations/1720532894811_ft_balances.js

```diff
@@ -0,0 +1,125 @@
+/** @param { import("node-pg-migrate").MigrationBuilder } pgm */
+exports.up = pgm => {
+  pgm.createTable('ft_balances', {
+    id: {
+      type: 'bigserial',
+      primaryKey: true,
+    },
+    address: {
+      type: 'text',
+      notNull: true,
+    },
+    token: {
+      type: 'text',
+      notNull: true,
+    },
+    balance: {
+      type: 'numeric',
```

Comment:
I'm assuming this is the raw number without the metadata decimals for each token, right?

Reply:
Yep that is correct